src/ble/l2cap/mod.rs

```diff
@@ -176,14 +176,14 @@ pub struct BleChannelMap<A: Attributes, S: SecurityLevel> {
     sm: SecurityManager<S>,
 }
 
-impl BleChannelMap<NoAttributes, NoSecurity> {
+impl<A: Attributes> BleChannelMap<A, NoSecurity> {
     /// Creates a new channel map with no backing data for the connected protocols.
     ///
     /// This means:
     /// * The attribute server on channel `0x0004` will host an empty attribute set.
-    pub fn empty() -> Self {
+    pub fn with_attributes(att: A) -> Self {
```

Comment:
I kind of want to keep `empty` in addition to this, just to offer a function that creates the minimum viable product (for `AttributeServer` it doesn't matter much, but this may be called from the outside).

Reply:
Sure thing :)